src/plugins/discover/public/application/main/components/layout/discover_documents.tsx

```diff
@@ -179,7 +180,7 @@ function DiscoverDocumentsComponent({
     [stateContainer]
   );
 
-  const onResizeDataGrid = useCallback(
+  const onResizeDataGrid = useCallback<NonNullable<UnifiedDataTableProps['onResize']>>(
```

Comment:
I also checked Dashboard page and it does not seem to offer resetting a column width. Should it?

Reply:
That was because the saved search embeddable didn't support `onResize` or `grid` modifications, but I added support for it now: https://github.com/elastic/kibana/pull/190288/commits/7b3583e1af1ebea49c99c0d105552e8d299d7509.